media/css/pebbles/components/_buttons-download.scss

```diff
@@ -123,38 +119,32 @@ ul.download-list {
     display: none !important;
 }
 
-// Uncomment this to enable the win64 download buttons:
-// .win7up.x86.x64 .download-button .os_win64,
 .linux .download-button .os_linux,
 .linux.x86.x64 .download-button .os_linux64,
 .windows .download-button .os_win,
 .osx .download-button .os_osx,
 .android .download-button .os_android,
 .download-button-android .os_android,
 .android .download-button-desktop .download-list,
-.android .download-button-desktop small.os_win,
```

Comment:
Note for reviewer: these selectors with `small.os_win` in them are from back when there was a list of links underneath the download buttons. These are now redundant